docs/release-process.md

```diff
@@ -29,26 +29,26 @@ Release shepherd responsibilities:
 * Perform releases (from first RC to actual release).
 * Announce all releases on all communication channels.
 
-| Release   | Time of first RC         | Shepherd (GitHub handle) |
-|-----------|--------------------------|--------------------------|
-| v0.22.0   | (planned) 2021.06.16     | TBD                      |
-| v0.21.0   | (planned) 2021.05.04     | TBD                      |
-| v0.20.0   | 2021.04.23               | `@kakkoyun`              |
-| v0.19.0   | 2021.03.02               | `@bwplotka`              |
-| v0.18.0   | 2021.01.06               | `@squat`                 |
-| v0.17.0   | 2020.11.18               | `@metalmatze`            |
-| v0.16.0   | 2020.10.26               | `@bwplotka`              |
-| v0.15.0   | 2020.08.12               | `@kakkoyun`              |
-| v0.14.0   | 2020.07.10               | `@kakkoyun`              |
-| v0.13.0   | 2020.05.13               | `@bwplotka`              |
-| v0.12.0   | 2020.04.15               | `@squat`                 |
-| v0.11.0   | 2020.02.19               | `@metalmatze`            |
-| v0.10.0   | 2020.01.08               | `@GiedriusS`             |
-| v0.9.0    | 2019.11.26               | `@bwplotka`              |
-| v0.8.0    | 2019.10.09               | `@bwplotka`              |
-| v0.7.0    | 2019.08.28               | `@domgreen`              |
-| v0.6.0    | 2019.07.12               | `@GiedriusS`             |
-| v0.5.0    | 2019.06.31               | `@bwplotka`              |
+| Release   | Time of first RC         | Shepherd (GitHub handle)    |
+|-----------|--------------------------|-----------------------------|
+| v0.22.0   | (planned) 2021.06.16     | TBD                         |
+| v0.21.0   | (planned) 2021.05.04     | `@metalmatze` and `@onprem` |
```

Comment:
This date is already in the past. Do we mean June?

Reply:
I am not sure if we want to modify that or not haha. It was correct for a while, but there were some delays in releases and we have drifted a bit from the planned dates,

Reply:
I would suggest to make a new planning. This provides value to users to know when releases are coming so they can act accordingly (i.e. test RC's or upgrade because they are really eager for a feature).